bigbluebutton-html5/imports/ui/components/app/service.js

```diff
@@ -105,6 +106,15 @@ const clearModal = () => {
   showModal(null);
 };
 
+function getCaptionsStatus() {
+  var CCEnabled = Storage.getItem('closedCaptions');
+  if(CCEnabled == true) {
```

Comment:
It can be undefined for the first time. I just wanted to make sure it always passes either true or false.